api/num_correct.py

```diff
@@ -8,10 +8,12 @@ def calculate_num_correct(param):
     student_list = {get_keyword_value("student_list"): get_student_list(inp)}
     sortedResponses = get_sorted_responses(student_list)
     numStudents = len(sortedResponses)
-    numItems = len (sortedResponses[0])
+    numItems = len(sortedResponses[0])
     idList = get_id_list(inp)
     num_correct_list = []
     num_correct_dict = {}
+    exclude_items = list(param.get(get_keyword_value("exclude_items"), []))
```

Comment:
What is this for?

Reply:
I was testing something. I will remove it.